netmanager/src/views/apis/faultDetection.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const GET_FAULTS = `${process.env.REACT_APP_BASE_URL}get-faults`;
```

Comment:
The vale of `REACT_APP_BASE_URL` is https://staging-platform.airqo.net/api/v1/ in the environment variables.